templates/cisco_ios_show_object-group.textfsm

```diff
@@ -0,0 +1,31 @@
+Value Required,Filldown OGR_TYPE (Service|Network)
```

Comment:
I'd probably just make these `TYPE/NAME`

Reply:
Can you potentially align around the naming such as similar template? https://github.com/networktocode/ntc-templates/blob/master/templates/cisco_asa_show_running-config_object_network.textfsm.

Not too many changes required to get this merged in and we appreciate your patience!

Reply:
```suggestion
Value Required,Filldown TYPE (Service|Network)
```
OK, I agree